environment.rb

```diff
@@ -0,0 +1,102 @@
+module Environment
+
+  class Depot
+    attr_accessor :packs
+
+    def initialize
+      @packs = {
+        simple_transportation_pack: {
+          intelligence: %i[cellphone],
+          items: %i[medipack chevy_versa]
+        },
+        standard_transportation_pack: {
+          intelligence: %i[cellphone antenna],
+          arsenal: [:colt_1911],
+          items: %i[handcuffs medipack chemestry chevy_versa]
+        },
+        simple_mission_pack: {
+          intelligence: %i[infopack laptop cellphone antenna],
+          arsenal: [:colt_1911],
+          items: %i[handcuffs medipack chemistry financial]
+        },
+        standard_mission_pack: {
+          intelligence: %i[infopack laptop cellphone antenna],
+          arsenal: %i[remington_870 colt_1911 machete hatchet],
+          items: %i[handcuffs medipack chemistry financial]
+        }
+      }
+    end
+
+  end
+
+  class Control
+    attr_accessor :missions
+    states = [:paused, :aborted, :failed, :accomplished]
+
+    def initialize
+      @missions = Hash.new
```

Comment:
Again, the Rubocop gem prefers the `@missions = {}` initialization.

Reply:
Alright, fixed it. TY for your reviews!